kql/ast.py

```diff
@@ -116,7 +119,7 @@ def _render(self):
                 escaped.append(self.escapes[char])
             else:
                 escaped.append(char)
-        return ''.join(escaped)
+        return json.dumps(''.join(escaped))
```

Comment:
why was dumps needed @Mikaayenson?

Reply:
It may be overkill but we want to ensure the result is quoted string per the issue.